website/docs/api/docusaurus.config.js.md

```diff
@@ -191,6 +191,7 @@ module.exports = {
   themeConfig: {
     colorMode: {
       defaultMode: 'light',
+      hideableSidebar: true, // default is false
```

Comment:
```suggestion
      hideableSidebar: false,
```

Thanks

Just putting `false` here as the values in this config actually are the default values ;)